charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -1590,6 +1590,9 @@ spec:
                                     tokenFile:
```

Comment:
We now need to update the MeteringConfig CRD values for the Prometheus connector now that we've decided on a general layout (then run `make metering-manifests` to re-generate the manifests.)

Reply:
updated in 7dc46406